### ISSUE TITLE
Data in DarkWeb

### ISSUE THREAD

**Issue by Ducthanh269** [LuatixHQ/opencti#9202]
Hi everyone, I want to ask about where data on the dark web, such as leaked accounts or organizational vulnerabilities that are being discussed on the dark web, will be stored in which section of OpenCTI and is there any external connector that can import data from the dark web.

**Comment by nino-filigran** [comment]
We're currently working on a spycloud connector: https://github.com/OpenCTI-Platform/connectors/issues/2563

**Comment by Ducthanh269** [comment]
I don't see SpyCloud Connector anywhere and now I have a list of exposed account information (link, User/Password), which part of OpenCTI should I import it into (Indicator, Observed Data, ...)?

**Comment by nino-filigran** [comment]
Yes it's normal, it's because the connector is not yet available, we're currently working on it and should be available for the 6.5 release (milestone)

**Comment by Ducthanh269** [comment]
Thanks for your information but i ask now i have information about the accounts displayed including (user,pass,url) how should i put it into OpenCTI and which part of STIX will it be. please explain this part clearly so i can defend my university thesis about OpenCTI. thank you very much

**Comment by nino-filigran** [comment]
f I understand well, you have already managed to get form a source some data related to some leaked users, passwords & urls. And your question is about how to modelize his inforation in OpenCTI, right?

I don't know about the service & API you use on your side but to give you an idea, the way we paln to modelize data with spycloud is: 
- for each record, we would create an incident. 
- then depending on the information we will create an observable. In your case, I would suggest creating: 
   - users -> user account in OCTI
   -  password -> credentials in OCTI
   - URL -> URL in OCTI

And for each obserrvable you create, create a link Observable (User account/credential/URL) RELATED TO Incident, so that for each record you'll have, you will have an entity that is related, helping you to potentially pivot & investigate. Hope that can help.
Of course, this modelization depends on the service you're using, the API response.

**Comment by ishakarslan** [comment]
I reach out to their team to get an api key for connector, they direcly told me "Just to be transparent our solutions typically start at around $50K per year", which may not be feasible for smaller setups or early-stage integrations.

**Comment by alice-debra** [comment]
Hello @Ducthanh269 @ishakarslan,
We’re following up to see if you still need an answer to this question. If you still need help with this, don't hesitate to ask on our Slack: https://filigran-community.slack.com/, or tell me what you need :)
Kind regards,
Alice

**Comment by alice-debra** [comment]
Hello,
Due to a lack of recent activity on this issue, we will proceed to close it for now. We sincerely thank you for your feedback. Please don’t hesitate to reopen the issue if you still have any questions.
Kind regards,
Alice
